src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
```

Comment:
It's better to destructure your "person" with variables like age, name, etc. This will help you to avoid writing extra code like "person.name"  each time below.